frontend/packages/core/src/Table/table.tsx

```diff
@@ -122,6 +125,8 @@ interface TableProps extends Pick<MuiTableProps, "stickyHeader"> {
   actionsColumn?: boolean;
   /** Make table responsive */
   responsive?: boolean;
```

Comment:
is this not needed here anymore as well?

Reply:
It's still being picked from the TableProps